cinema/serializers.py

```diff
@@ -1 +1,91 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Actor, CinemaHall, Genre, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+    def get_full_name(self, obj):
+        return f"{obj.first_name} {obj.last_name}"
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    capacity = serializers.IntegerField(read_only=True)
+
+    class Meta:
+        model = CinemaHall
+        fields = ["id", "name", "rows", "seats_in_row", "capacity"]
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    genres = GenreSerializer(many=True, read_only=True)
+    actors = ActorSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+
+class MovieCreateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
```

Comment:
The `MovieCreateSerializer` includes `genres` and `actors` fields, but these are marked as read-only in the `MovieSerializer`. If you intend to allow creation or update of these fields, you need to handle them appropriately, possibly using `PrimaryKeyRelatedField` or similar fields that support write operations.